<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1244" started-at="2018-06-11T01:30:46Z" finished-at="2018-06-11T01:30:47Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1244" started-at="2018-06-11T01:30:46Z" finished-at="2018-06-11T01:30:47Z">
      <class name="AdbApiTesting">
        <test-method status="PASS" signature="hello()[pri:0, instance:AdbApiTesting@7c0e2abd]" name="hello" duration-ms="1203" started-at="2018-06-10T18:30:46Z" finished-at="2018-06-10T18:30:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hello -->
        <test-method status="FAIL" signature="helloWithId()[pri:0, instance:AdbApiTesting@7c0e2abd]" name="helloWithId" duration-ms="18" started-at="2018-06-10T18:30:47Z" finished-at="2018-06-10T18:30:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Hello  kavin] but found [Hello1  kavin]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Hello  kavin] but found [Hello1  kavin]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at AdbApiTesting.helloWithId(AdbApiTesting.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- helloWithId -->
      </class> <!-- AdbApiTesting -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
